alacritty_terminal/src/vi_mode.rs

```diff
@@ -9,7 +10,8 @@ use crate::term::cell::Flags;
 use crate::term::Term;
 
 /// Possible vi mode motion movements.
-#[derive(ConfigDeserialize, Debug, Copy, Clone, PartialEq, Eq)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub enum ViMotion {
```

Comment:
You're changing the serialization behavior by switching from `ConfigDeserialize` to `Deserialize` here.

This will no longer support case-insensitive parsing. I do not see any wrapper in `alacritty`.

Reply:
I've changed it to something ugly, but at least it doesn't depend on `toml` anymore...

Reply:
Changed to it `rename_all = "lowercase"`